Scenes/Player/Player.tscn

```diff
@@ -1,18 +1,133 @@
-[gd_scene load_steps=4 format=2]
+[gd_scene load_steps=26 format=2]
 
-[ext_resource path="res://icon.png" type="Texture" id=1]
+[ext_resource path="res://Spritesheets/clown_000.png" type="Texture" id=1]
 [ext_resource path="res://Scenes/Player/Player.gd" type="Script" id=2]
 [ext_resource path="res://Scenes/Bomb/Bomb.tscn" type="PackedScene" id=3]
 
+[sub_resource type="AtlasTexture" id=6]
+atlas = ExtResource( 1 )
+region = Rect2( 0, 192, 32, 32 )
+
+[sub_resource type="AtlasTexture" id=7]
+atlas = ExtResource( 1 )
+region = Rect2( 32, 192, 32, 32 )
+
+[sub_resource type="AtlasTexture" id=8]
+atlas = ExtResource( 1 )
+region = Rect2( 64, 192, 32, 32 )
+
```

Comment:
To trzeba będzie usunąć z poziomu edytora (dać mu plik PlayerAnimation.tres w frames)